bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,539 @@
+
+# WorxLandroid Binding
+
+This is the binding for Worx Landroid robotic lawn mowers. 
+It connects openHAB with your WorxLandroid mower using the API and MQTT. 
+This binding allows you to integrate, view and control supported Worx lawn mowers in the openHAB environment.
+
+## Supported Things
+
+Currently following Things are supported:
+
+- `bridge`: **Bridge Worx Landroid API** Thing representing the handler for Worx API
+- `mower`: One or many Things for supported **Landroid Mower**'s
+
+## Discovery
+
+A Bridge is required to connect to the Worx API. 
+Here you can provide your credentials for your WorxLandroid account. 
+Once the Bridge has been added Worx Landroid mowers will be discovered automatically.
+
+## Things Configuration
+
+The following options can be set for the `bridge`:
+
+| Property | Description                              |
+|----------|------------------------------------------|
+| username | Username to access the WorxLandroid API. |
+| password | Password to access the WorxLandroid API. |
+
+
+The following options can be set for the `mower`:
+
+| Property              | Description                                                                                  | Default | Advanced |
+|-----------------------|----------------------------------------------------------------------------------------------|---------|----------|
+| serialNumber          | Serial number of the mower                                                                   |      -  |   No     |
+| refreshStatusInterval | Interval for refreshing mower status (ONLINE/OFFLINE) and channel 'common#online' in seconds |   3600  |  Yes     |
+| pollingInterval       | Interval for polling in seconds (min="30" max="7200").                                       |   1200  |  Yes     |
+
+
+Default values for `refreshStatusInterval` and `pollingInterval` are the recommended settings in order to prevent a 24h ban from Worx.
+Lower polling and refresh values will likely result in a 24h ban for your account.
+
+## Channels
+
+Currently following **Channels** are supported on the **Landroid Mower**:
+
+##### common
```

Comment:
Please adjust headers (capitalizxed and level)
```suggestion
### Common
```

Reply:
Corrected